typing/btype.mli

```diff
@@ -69,8 +69,18 @@ val row_field: label -> row_desc -> row_field
         (* Return the canonical representative of a row field *)
 val row_more: row_desc -> type_expr
         (* Return the extension variable of the row *)
+
+val is_fixed: row_desc -> bool
+(* Return whether the row is directly marked as fixed or not *)
 val row_fixed: row_desc -> bool
-        (* Return whether the row should be treated as fixed or not *)
+(* Return whether the row should be treated as fixed or not.
+   In particular, [is_fixed row] implies [row_fixed row].
+*)
+val fixed_explanation: row_desc -> fixed_explanation option
+(* Return the potential explanation for the fixed row *)
+val merge_fixed_explanation: row_desc -> row_desc -> fixed_explanation option
+(* Merge two explanations for a fixed row *)
```

Comment:
Since you're already not respecting the same style as in the rest of this file (i.e. indenting the comments by like 36c), perhaps you could also add blank lines in between items?

:smiley:

Reply:
You are right, let's add some breathing space.